src/dotnet-bootstrapper/Commands/Search/SearchCommand.cs

```diff
@@ -43,10 +43,15 @@ public override int Execute()
 
             foreach (ProductRelease release in releases)
             {
+                // Get release.Sdks latest version
+                var latestSdk = release.Sdks
+                    .OrderByDescending(sdk => sdk.Version)
+                    .FirstOrDefault();
+
                 productMetadataTable.AddRow(
                     release.Version.ToString(), 
                     release.ReleaseDate.ToString("yyyy-MM-dd"), 
```

Comment:
This forces the US approach for writing dates. Output, even from the CLI should be considered as a UI I think and use the UI thread culture for date formatting. This would get flagged under globalization during 1CS. Consider using the shortformat ("d") specifier. You should be able to do something like `ToString("d", CultureInfo.CurrentUICulture)`